packages/sources/svelte/src/components/indicators/VtmnProgressbar/VtmnProgressbarCircular.svelte

```diff
@@ -81,7 +81,13 @@
     <img class="vtmn-progressbar_image" src={img} alt={imgAlt} />
   {/if}
   {#if label && !indeterminate}
-    <span class="vtmn-progressbar_label" data-value={progress} />
+    <span class="vtmn-progressbar_label" aria-live="assertive">{progress}%</span
+    >
+  {/if}
+  {#if indeterminate}
+    <span id={$$restProps['aria-labelledby'] || undefined} class="vtmn-sr-only"
```

Comment:
Just restProps['aria-labelledby'] should work :)

Reply:
Same as for React, I don't want an undefined id if not aria-label is passed